README.md

```diff
@@ -30,97 +29,109 @@ yarn lint
 ```
 
 ## Test debugging
+
 For visualizing and debugging tests in VS code install following two extensions:
 
 [Test Explorer UI](https://marketplace.visualstudio.com/items?itemName=hbenl.vscode-test-explorer)
 
 [Mocha Test Explorer](https://marketplace.visualstudio.com/items?itemName=hbenl.vscode-mocha-test-adapter)
 
-
 ## Deploying to remote networks
-Requirement: `gcloud` must be set up and user account must have access staked-celo-alfajores and staked-celo-staging on GCP. 
+
+Requirement: `gcloud` must be set up and user account must have access staked-celo-alfajores and staked-celo-staging on GCP.
 Next, ensure the environment variables have been decrypted, using `yarn keys:decrypt`.
 Then use the following commands to deploy depending on the desired target environment.
-For example, the below command will deploy to the Alfajores network, using the decrypted private key, the default Alfajores rpc url and the other variables in `.env.alfajores`. 
+For example, the below command will deploy to the Alfajores network, using the decrypted private key, the default Alfajores rpc url and the other variables in `.env.alfajores`.
 
-Alfajores : 
-```
+Alfajores:
+
+```bash
 yarn deploy --network alfajores
 ```
 
 You may immediately verify the deployment, with the following commands.
 
-Alfajores : 
-```
+Alfajores :
+
+```bash
 yarn verify:deploy --network alfajores
 ```
 
 ## Verify on CeloScan
+
 1. Get CeloScan api key
-1. Update api key in hardhat.config.ts (etherscan.apiKey)
-1. Get constructor arguments of deployed smart contract
-  * Find contract in `deployments/[network]` (example deployments/celo/MultiSig_Proxy.json)
-  * In root level there are constructur arguments in `args`
+2. Update api key in hardhat.config.ts (etherscan.apiKey)
```

Comment:
I guess it's nice to have -> but the actual instruction should be to:
- add setup section before `Testing` with two steps below
- perform `cp .env.example .env`
- fill in api key inside `.env` file (since dotenv is present in `hardhat.config.ts`)